sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2564,11 +2564,6 @@ pi_result piMemBufferCreate(pi_context Context, pi_mem_flags Flags, size_t Size,
                             void *HostPtr, pi_mem *RetMem,
                             const pi_mem_properties *properties) {
 
-  // TODO: implement read-only, write-only
-  // if ((Flags & PI_MEM_FLAGS_ACCESS_RW) == 0) {
```

Comment:
Can you please return the check for the expected/"supported" values in flags, and die if somethin else is passed, so we know some semantics is potentially missed.

Reply:
done